### ISSUE TITLE
Support Azure

### ISSUE THREAD

**Issue by tkennes** [camptocamp/terraboard#289]
Hi there,

Maybe I'm missing something, but it seems that only GCP and AWS are supported. Is that true?

**Comment by davidkarlsen** [comment]
Yes, a state-provider for Azure needs to be developed to support this.
